src/experiments.js

```diff
@@ -66,6 +66,16 @@ export function isCanary(win) {
   return !!(win.AMP_CONFIG && win.AMP_CONFIG.canary);
 }
 
+/**
+ * Returns binary type, e.g., canary, control, or production.
+ * @param {!Window} win
+ * @return {string}
+ */
+export function getBinaryType(win) {
+  return win.AMP_CONFIG && win.AMP_CONFIG.type
```

Comment:
nit: IIRC, the style guide suggests that line breaks for the ternary operator appear after the `?`. Also, parenthesize the boolean expression for clarity.

```
return (win.AMP_CONFIG && win.AMP_CONFIG.type) ?
    win.AMP_CONFIG.type : 'unknown';
```